documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,136 @@
+### Before RC1
+- [ ] (network upgrade) Make sure all [Lotus dependencies are updated to the correct versions for the network upgrade](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/Update_Dependencies_Lotus.md)
+   - Link to Lotus PR:
+- [ ] Open PR against [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) with title `docs(vX.Y.Z): release template improvements` for improving future releases.
+   - Link to PR:  
+   - This will get merged in a `Post Release` step, but improvements are better done by collecting notes along the way rather than just thinking about it at the end.
+- [ ] Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release-related changes to this branch.
+- `master` branch Version string updates
+   - [ ] bump the version(s) in `build/version.go` to `vX.Y.(Z+1)-dev`. 
+      - Ensure to update the appropriate version string based on whether you are creating a node release (`NodeBuildVersion`), a miner release (`MinerBuildVersion`), or both.
+   - [ ] Run `make gen && make docsgen-cli` before committing changes.
+   - [ ] Create a PR with title `build(vX.Y.Z+1): set initial version string`
+     - Link to PR: 
+   - [ ] Merge PR
+
+### RCs
+
+[//]: # (Copy/paste this "RCX" section for each additional RC, and increment "X")
+#### RCX
+> [!IMPORTANT]
+> These PRs should be done in and target the `release/vX.Y.Z` or `release/miner/vX.Y.Z` branch.
+
+**Backport PR**
+
+[//]: # (For RC1 there likely isn't any backporting to do and thus no PR which reduces the steps.)
+[//]: # (We do need all these steps for RC2 onwards though.)
+[//]: # (If steps are removed for the RC1 checklist, they need to be preserved for future RCs/stable.)
+[//]: # (For RC1 we still need to make sure the tracked items land though.)
+- [ ] All explicitly tracked items from `Dependencies for releases` have landed
+- [ ] Backported [everything with the "backport" label](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+) 
+- [ ] Removed the "backport" label from all backported PRs (no ["backport" issues](https://github.com/filecoin-project/lotus/issues?q=label%3Arelease%2Fbackport+))
+- [ ] Create a PR with title `build(vX.Y.Z): backport changes for vX.Y.Z-rcX`
+   - Link to PR: 
+- [ ] Merge PR 
+
+**Release PR**
+
+- [ ] Update the version string(s) in `build/version.go` to one ending with '-rcX'. 
+    - Ensure to update the appropriate version string based on whether you are creating a node release (`NodeBuildVersion`), a miner release (`MinerBuildVersion`), or both.
+- [ ] Run `make gen && make docsgen-cli` to generate documentation
+- [ ] Changelog prep
+   - [ ] Generate changelog using the script at `scripts/mkreleaselog`
```

Comment:
I think we can remove this step since we now get the changelog automatically generated for the release/tag when we open a PR that changes the version string: https://github.com/filecoin-project/lotus/pull/12096.

One problem that arises is how we populate this back into the CHANGELOG.md file in Lotus, though.

Reply:
Oh really - that doesn't seem good/right.  I assume we want the text for the release notes to be under code review, which means its source of truth should be changelog.  I'll look into this more as thought we hit this recently with one of the other repos as well.

Reply:
Oh doh - your comment is about just this line of `scripts/mkreleaselog`.  My bad.  I misread this in how it was coming through in the GitHub conversation view (not files view).  I agree this isn't needed anymore: https://github.com/filecoin-project/lotus/blob/master/.github/workflows/release.yml#L256 .  I'll remove it now.

Reply:
commit with adjust changelog prep steps: https://github.com/filecoin-project/lotus/pull/12356/commits/e3805466b376aa56d006cf471adf4c1d131fc384

I also added commands there that can run for looking at the history to help identify if anything missing in the changelog.  These were just ideas I would do off the top of my head but maybe you had other commands you'd run?